packages/store/tests/state-token.spec.ts

```diff
@@ -40,8 +40,7 @@ describe('[TEST]: StateToken', () => {
         template: '{{ myState$ | async | json }}'
       })
       class MyComponent {
-        @Select(TODO_LIST_TOKEN)
-        myState$: Observable<string[]>;
+        myState$: Observable<string[]> = inject(Store).select(TODO_LIST_TOKEN);
```

Comment:
Maybe this is cleaner?
Use the store reference that we already have.
```suggestion
        myState$ = this.storeApp.select(TODO_LIST_TOKEN);
```